src/test/datascience/helpers.node.ts

```diff
@@ -99,6 +102,15 @@ export async function insertIntoInputEditor(source: string) {
     return vscode.window.activeTextEditor;
 }
 
+export async function setActiveInterpreter(
+    apiProvider: IPythonApiProvider,
+    resource: Resource,
+    interpreter: vscode.Uri
+) {
+    const pythonApi = await apiProvider.getApi();
+    return pythonApi.setActiveInterpreter(getFilePath(interpreter), resource);
```

Comment:
Make sure you're also clearing Python's workspace interpreter setting in the teardown, Otherwise the interpreter setting that we use will be ignored

Reply:
Ah good point.